client/packages/system/src/Item/Components/ItemVariantSelector/columns.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {
+  ColumnDescription,
+  RadioCell,
+  TooltipTextCell,
+  useColumns,
+} from '@openmsupply-client/common';
+import { ItemVariantFragment } from '../../api';
+
+interface ItemVariantSelectorColumnProps {
+  selectedId: string | null;
+  onVariantSelected: (itemVariantId: string | null) => void;
+  displayInDoses: boolean;
+}
+
+export const useItemVariantSelectorColumns = ({
+  selectedId,
+  onVariantSelected,
+  displayInDoses,
+}: ItemVariantSelectorColumnProps) => {
+  const columnDefinition: ColumnDescription<ItemVariantFragment>[] = [
+    {
+      key: 'itemVariantSelector',
+      Cell: props => (
+        <RadioCell
+          {...props}
+          selectedId={selectedId}
+          onSelected={onVariantSelected}
+          groupName="item-variant-selector"
+        />
+      ),
+      accessor: ({ rowData }) => rowData.id,
+      width: 50,
+    },
+    [
+      'name',
+      {
+        Cell: TooltipTextCell,
+        width: 300,
+      },
+    ],
+  ];
+
+  if (displayInDoses) {
```

Comment:
Wasn't sure if we just hiding for non-vaccine items, or if should be using pref + checking vaccine... have implemented this in the variant tab too... assuming latter because we're doing it as part of the doses management work, but please correct me if I'm wrong @jmbrunskill @mark-prins @DhanyaHerath

Reply:
I'm not 100% sure, but I think how you have it makes sense. E.g.
You need both `Display vaccines in doses` turned on and to be looking at a vaccine item to see this.